api/api.yaml

```diff
@@ -47,6 +47,24 @@ paths:
       tags:
         # Put in facets_controller.py instead of default_controller.py.
         - Facets
+  /exportUrl:
+    # This is post because this is not idempotent; a GCS file is created.
+    post:
+      description: >
+        Gets a signed URL to a GCS zip file of JSON files. The JSON files
+        represent entites to be exported to a Saturn workspace.
+        https://bvdp-saturn-prod.appspot.com/#import-data will be called with
```

Comment:
The API documentation should define what the API does instead of speculating what the client should or shouldn't do with the returned results. Lines 56-59 are true, but that documentation probably belongs on the UI side.

Reply:
Done